docs/developer/extending/apps/subscription-webhook-payloads.mdx

```diff
@@ -23,16 +24,17 @@ building a GraphQL query you will be able to use prompts and build the whole sub
 Inside the query, you must define the event type for which the payload will be generated. The event type must be supported by our subscription mechanism (see the [list of supported events](developer/api-reference/unions/event.mdx)).
 
 :::warning
-Only asynchronous webhooks are able to use subscriptions at the moment. Synchronous webhooks will be supported in the future.
+When defining subscription payload for synchronous webhooks some fields are blocked to prevent circular calls.
+More info on which fields are not allowed can be found in Synchro0nous
```

Comment:
> can be found in Synchro0nous

???

Reply:
🚓

Reply:
oops... this was supposed to be deleted. thx.